index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+            <li>
+                <a href="#welcome-section">Welcome</a>
+            </li>
+            <li>
+                <a href="#projects">Check my projects</a>
+            </li>
+            <li> <a href="#contact">Contact me</a>
+            </li>
+        </ul>
+    </nav>
+    <section id="welcome-section">Welcome!
+        <H1> Welcome to my personal Portfolio</H1>
+    </section>
+    <section id="projects">
+        <p><h2> Here are some of my projects:</h2></p>
+        <div>
+            
+            <span id="project-header">
+            <p class="project-title"> Project #1</p>  
+              
+            </span>

```

Comment:
Here it is better to indent the p tag it will make your code more readable.  And also remove unnecessary blank lines.

Reply:
Great review and clear comment. Well done